components/scream/src/physics/rrtmgp/atmosphere_radiation.cpp

```diff
@@ -144,6 +144,7 @@ void RRTMGPRadiation::set_grids(const std::shared_ptr<const GridsManager> grids_
   add_field<Computed>("sfc_flux_dif_vis", scalar2d_layout, Wm2, grid_name);
   add_field<Computed>("sfc_flux_sw_net" , scalar2d_layout, Wm2, grid_name);
   add_field<Computed>("sfc_flux_lw_dn"  , scalar2d_layout, Wm2, grid_name);
+  add_field<Computed>("cosine_solar_zenith_angle",scalar2d_layout,nondim,grid_name);
```

Comment:
Has this been added as a "computed" field for a reason?  Was it just to be able to see it as output in debugging?  If so, do we want to keep it like this or put it back in the mem buffer?

Reply:
I wanted to be able to output it. It's an output from EAM as well, so allows comparing with SCREAMv0/EAM. We probably won't regularly look at it, but it was pretty useful to have for this exercise. We can revert it back to a non-tracked field in the mem buffer if you think that's better. I don't think it will impact performance either way.

Reply:
Pushed a fix. I don't think it helps mem right now, but would if we were to use a unified buffer. No sense polluting the FM with fields we won't use anyways.